tests/fixtures/music_nerd/beatles_with_history.hocon

```diff
@@ -0,0 +1,49 @@
+# Copyright (C) 2023-2025 Cognizant Digital Business, Evolutionary AI.
+# All Rights Reserved.
+# Issued under the Academic Public License.
+#
+# You can be released from the terms, and requirements of the Academic Public
+# License by purchasing a commercial license.
+# Purchase of a commercial license is mandatory for any use of the
+# neuro-san SDK Software in commercial settings.
+#
+# END COPYRIGHT
+
+# This is file defines everthing necessary for a data-driven test.
+{
+    # Describes what agent to test against.
+    "agent": "music_nerd",
+
+    # Interactions are a series of dictionaries with request elements paired with
+    # descriptions of response checks.
+    "interactions": [
+        {
+            # This is what we send as input to streaming_chat()
+            "text": "Who did Yellow Submarine?",
```

Comment:
Currently we are only specifying text, but in short order the future we will specify other aspects of the ChatMessage such as sly_data.